### PR TITLE
Create rds-create-freestoragespace-alarms

### PR DESCRIPTION
**Pull request by mpavlisin**
https://app.asana.com/0/20810103798902/31039634612370

Creating rds-create-freestoragespace-alarms.

### CODE REVIEW

**Comment by lra** [comment]
A lot of those files are executables. They should not.

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  pre:
+    - pip install flake8
+
+test:
+  override:
+    - flake8:
+        parallel: true
+        files:
+          - rds_create_freestoragespace_alarms/**/*.py
+    - rds-create-freestoragespace-alarms --help
```
Comment (lra) [inline circle.yml:11]: not sure it will work if you don't install it

File: rds_create_freestoragespace_alarms/main.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+"""rds_create_freestoragespace_alarms
```
Comment (lra) [inline rds_create_freestoragespace_alarms/main.py]: The script name once installed is `rds-create-freestoragespace-alarms`

File: rds_create_freestoragespace_alarms/main.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+Script used to create a below 20 pct. Low-FreeStorageSpace alarm
+in AWS CloudWatch for all RDS instances
+
+Usage:
+    rds_create_freestoragespace_alarms [options]
```
Comment (lra) [inline rds_create_freestoragespace_alarms/main.py]: rds-create-freestoragespace-alarms

File: rds_create_freestoragespace_alarms/main.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    rds_create_freestoragespace_alarms [-h | --help]
```
Comment (lra) [inline rds_create_freestoragespace_alarms/main.py]: rds-create-freestoragespace-alarms

File: circle.yml (hunk 1 above)
Comment (lra) [inline circle.yml:11]: my bad, it does install it

**Comment by lra** [comment]
files are still executables.
read about permissions and make those file non executables.

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,16 @@
+develop:
+	python setup.py develop
+
+undevelop:
+	python setup.py develop --uninstall
+
+test:
+	flake8 rds-create-freestoragespace-alarms
```
Comment (lra) [inline Makefile]: flake8 requires a path. `rds-create-freestoragespace-alarms` is not a valid path

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+clean:
+	rm -rf rds-create-freestoragespace-alarms.egg-info/
```
Comment (lra) [inline Makefile:11]: not a valid path

File: README.md [hunk 7]
```diff
@@ -2,4 +2,26 @@
 
 [![Circle CI](https://circleci.com/gh/percolate/rds-create-freestoragespace-alarms.svg?style=svg)](https://circleci.com/gh/percolate/rds-create-freestoragespace-alarms)
 
-Automate the creation of RDS FreeStoragepace Alarms
+Automate the creation of RDS Freestoragespace Alarms.
+The `Low-Freestoragespace` lower-bound limit in the script is 20%, but this can be altered.
+
+# Quick Start
+```bash
+"""rds_create_freestoragespace_alarms
```
Comment (lra) [inline README.md]: the command uses `-`

File: README.md [hunk 8]
```diff
@@ -2,4 +2,26 @@
...
+
+Script used to create a below 20 pct. Low-FreeStorageSpace alarm
+in AWS CloudWatch for all RDS instances
+
+Usage:
+    rds_create_freestoragespace_alarms [options]
```
Comment (lra) [inline README.md]: rds-create-freestoragespace-alarms

File: README.md [hunk 9]
```diff
@@ -2,4 +2,26 @@
...
+    rds_create_freestoragespace_alarms [-h | --help]
```
Comment (lra) [inline README.md]: rds-create-freestoragespace-alarms

**Comment by mpavlisin** [comment]
@lra thanks. I got a lot of learning to do here

File: Makefile (hunk 5 above)
Comment (mpavlisin) [inline Makefile]: @lra How are the tests passing? I have put the same over at the other repo

Comment (mpavlisin) [inline Makefile]:
``` bash
➜  rds-create-freestoragespace-alarms git:(import-all) flake8 rds_create_freestoragespace_alarms/**/*.py
➜  rds-create-freestoragespace-alarms git:(import-all)
```

File: Makefile (hunk 6 above)
Comment (mpavlisin) [inline Makefile:11]:
``` bash
➜  rds-create-freestoragespace-alarms git:(import-all) rm -rf rds-create-freestoragespace-alarms.egg-info/
➜  rds-create-freestoragespace-alarms git:(import-all)
```

Comment (lra) [inline Makefile:11]: `-f` means `don't yell if it does not exist.

File: Makefile (hunk 5 above)
Comment (lra) [inline Makefile]: I don't know, I have no time to dig, but you certainly don't want to keep something if you don't understand why it's working.

Comment (lra) [inline Makefile]: You wrote `rds-create-freestoragespace-alarms git:(import-all) flake8 rds_create_freestoragespace_alarms/**/*.py` and in circle.yml you wrote `flake8 rds-create-freestoragespace-alarms`.

`-` vs `_`.

**Comment by lra** [comment]
cool

**Comment by mpavlisin** [comment]
Thanks - i will register this in pip

On Wed, Aug 12, 2015 at 3:52 PM, Laurent Raufaste notifications@github.com
wrote:

> Merged #1
> https://github.com/percolate/rds-create-freestoragespace-alarms/pull/1.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/percolate/rds-create-freestoragespace-alarms/pull/1#event-380720468
> .

## 

Mihailo Pavlisin
Infrastructure Engineer
See what's new on the Percolate blog: blog.percolate.com

**Pull request merged**